larq/optimizers_test.py

```diff
@@ -90,6 +99,35 @@ class TestBopOptimizer:
     def test_bop_accuracy(self):
         _test_optimizer(lq.optimizers.Bop(fp_optimizer=tf.keras.optimizers.Adam(0.01)))
 
+    def test_bop_lr_scheduler(self):
```

Comment:
I think we should also add a test for TensorFlow 1.14+  that tests the usage with `tf.keras.optimizer.schedules` similar to what TensorFlow does for their optimizers. But this can be done in a future PR.